test/unit_tests/braket/experimental/autoqasm/test_operators.py

```diff
@@ -640,3 +727,30 @@ def test_control_flow():
 h __qubits__[0];"""
 
     assert test_control_flow().to_ir() == expected
+
+
+def test_py_assert() -> None:
+    """Test Python assertions inside an AutoQASM program."""
+
+    @aq.main
+    def test_assert(value: bool):
+        assert value
+
+    test_assert(True)
+    with pytest.raises(AssertionError):
+        test_assert(False)
+
+
+def test_py_list_ops() -> None:
+    """Test Python list operations inside an AutoQASM program."""
+
+    @aq.main
+    def test_list_ops():
```

Comment:
This is what `autoqasm/operators/data_structures.py` handles? It would be neat to see a short demo of how the transpiler/converter ties this to those definitions